imports/plugins/core/ui/client/providers/dragDropProvider.js

```diff
@@ -1,27 +1,11 @@
-import React, { Component, PropTypes, Children } from "react"; // eslint-disable-line
+import { Component, Children } from "react";
+import PropTypes from "prop-types";
 import { DragDropManager } from "dnd-core";
 import HTML5Backend from "react-dnd-html5-backend";
 import { registerComponent } from "@reactioncommerce/reaction-components";
 
 const defaultManager = new DragDropManager(HTML5Backend);
 
-// /**
```

Comment:
Is this comment no longer relevant?

Reply:
No, as it applied to that commented out function below it.